static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -803,6 +802,33 @@ $quote-mark: 35px;
         }
     }
 }
+
+// **************** Liveblog overides ***********
```

Comment:
Would it make sense to put these styles in `[live-blog.head.scss](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/content-garnett/live-blog/_live-blog.head.scss)`? Feels like `types.scss` ought to be used for generic styles that apply to all content types, rather than overrides for specific content types

Reply:
@SiAdcock If thats what has been decided going forward of course. But this is what seems to have been done before is use this to contain the overrides rather than the base styles.